build.sbt

```diff
@@ -1,22 +1,48 @@
-import Dependencies._
+import Dependencies.*
+
+// Your next release will be binary compatible with the previous one,
+// but it may not be source compatible (ie, it will be a minor release).
+ThisBuild / versionPolicyIntention := Compatibility.BinaryCompatible
+
+//TODO: after 3.1.0 release - clear up versionPolicyIgnored section
+/*
+versionPolicyReportDependencyIssues ignored dependencies when compared to safe-akka 3.0.0.
+All of those should not affect the library users, binary compatibility should be preserved.
+ */
+ThisBuild / versionPolicyIgnored ++= Seq(
+  //com.evolutiongaming:executor-tools_2.13: missing dependency
+  "com.evolutiongaming" %% "executor-tools",
+  //com.google.protobuf:protobuf-java: missing dependency
+  "com.google.protobuf" % "protobuf-java",
+  //org.scala-lang.modules:scala-java8-compat_2.13:
+  //  incompatible version change from 0.9.0 to 1.0.0 (compatibility: early semantic versioning
+  "org.scala-lang.modules" %% "scala-java8-compat",
+  //org.typelevel:cats-macros_2.13: missing dependency
+  "org.typelevel" %% "cats-macros",
+)
```

Comment:
I suppose that with next version we will be able to remove these - let's add to above TODO ticket too

Reply:
It's covered by the TODO above, I believe.